src/app/personside/infotabs/ytelser/YtelserListeElement.tsx

```diff
@@ -78,6 +80,29 @@ function YtelserListeElement(props: Props) {
         );
     }
 
+    if (isSykepengerSpokelse(props.ytelse)) {
+        const fom = getSykepengerSpokelseIdDato(props.ytelse);
+
+        return (
+            <li key={getUnikSykepengerSpokelseKey(props.ytelse)}>
+                <VisMerKnapp
+                    key={getUnikSykepengerSpokelseKey(props.ytelse)}
+                    ariaDescription="Vis sykepenger fra spokelse"
+                    valgt={props.erValgt}
+                    linkTo={dypLenker.ytelser.link(props.ytelse)}
+                    umamiEvent={{
+                        name: trackingEvents.detaljvisningKlikket,
+                        data: { fane: 'ytelser', tekst: 'sykepenger fra spokelse' }
+                    }}
+                >
+                    <Undertittel tag="h3">Sykepenger fra spokelse</Undertittel>
```

Comment:
Fjern "fra spokelse", dataen kjem både frå Speil og Infotrygd, så me kan berre kalle det for sykepenger